packages/manager/cypress/support/util/cloudpulse.ts

```diff
@@ -0,0 +1,54 @@
+// Function to generate random values based on the number of points
+
+import type { CloudPulseMetricsResponseData } from '@linode/api-v4';
+export const generateRandomMetricsData = (
+  time:
+    | 'Last 7 Days'
+    | 'Last 12 Hours'
+    | 'Last 24 Hours'
+    | 'Last 30 Days'
+    | 'Last 30 Minutes',
```

Comment:
```suggestion
   Labels,
```
reuse Labels from time range select component https://github.com/ACLPManager/manager/blob/aclp_develop/packages/manager/src/features/CloudPulse/shared/CloudPulseTimeRangeSelect.tsx